File: Day 5/SortAnArray.js

const arr = [10, 2, 5, 3, 1];

// Sort the array in ascending order.
array.sort();

console.log(array); // [1, 2, 3, 5, 10]

// Sort the array in descending order.
array.sort(function(a, b) {
  return b - a;
});

console.log(array); // [10, 5, 3, 2, 1]

File: Day 5/SortAnArray.py
# sort in ascending order 
arr = [10, 2, 5, 3, 1]
arr.sort()
print(arr)



# sort in descending order 
def sortArrDesc(arr):
"""Sorts an array in descending order."""
  for i in range(len(arr)):
    for j in range(i + 1, len(arr)):
      if arr[i] < arr[j]:
        arr[i], arr[j] = arr[j], arr[i]


arr = [10, 1, 5, 3, 2]
sortArrDesc(array)
print(array)
